meetings/2023-04-05.md

```diff
@@ -47,23 +54,99 @@
+* SC: I read "recommend a set of practices" as "best practices or guidelines". If it were requirements it'd be a deliverable, but this seems to be BP&G. 
+* SC: In the Charter "out of scope" has two parts: definition of identify mechanisms and definition of linked data formats. The part about set of practices is a very broad thing being thrown in. I'm not sure if we need to recommend best practices around AuthN or identity where, for example, we're also thinking those are out of scope. If we do anything about identity mechanisms, we could do best practices, but this seems to be reaching to far.
+* eP: I'm not sure if the intention wasn't to somehow mention that we'd try to build on other standards. I guess it might have been formulated in this way to have flexibility around normative requirements or recommendations but not committing to requirements. 
+* SC: May I ask eP to chime in in this issue? For us to translate this issue into a PR we need to remove some ambiguities. If you could say in what ways something would be within a practice and which ways out of scope, we can update the language. Right now it is virtually everything we do in Solid and I don't think that's the intention of the Charter.
+* JZ: Not directly on this point, but when you say identity and mention other several large topics out of scope, is there a place within Solid where that's a part of? 
+* SC: Good question. That is what we're doing in the CG. For example WebId-Profile falls under that category. It's a client-client spec, and we need to incubate that. What we are saying in the Charter is they are not intended to be part of it or in scope. Both in terms of client-client and also being an approach towards identity. 
+* JZ: I'm thinking of a conversation I had recently with someone who has big ideas on infrastructure and the relationship between an URI and identity. Not client-client and not even server-client, but something that Solid needs to be thinking about.
+* SC: That's under the CG. New material or things that needs to be conceptually worked out, spec'ed, incubated, or needs adoption falls under CG, in general. For example, if we had a lot of implementations with a lot of user adoption out there for WebID-Profile spec, or Type Indexes, we could make an argument that we have a good sense of how it works but needs to be spec'ed, so we'd need it to be part of the Charter, but we're not there yet. The purpose of CG is to move us closer to that so that it can be picked up by WG. Hope that clarifies.
+* JZ: Yes. WG has a specific charter and CG does everything else. 
```

Comment:
```suggestion
* JZ: I'm thinking of a conversation I had recently with someone who has big ideas on infrastructure and the relationship between an URI and identity. Not client-client and not even server-client, but something that Solid needs to be thinking about.
* SC: That's under the CG. New material or anything that needs to be conceptually worked out, spec'ed, incubated, or needs adoption falls under CG, in general. For example, if we had a lot of implementations with a lot of user adoption out there for WebID-Profile spec, or Type Indexes, we could make an argument that we have a good sense of how it works but it needs to be spec'ed, so we'd need it to be part of the Charter, but we're not there yet. The purpose of CG is to move things closer to that so that they can be picked up by WG. Hope that clarifies.
* JZ: Yes. WG has a specific charter and CG does everything else.
```